Personel/Adesh/7sep/sumOfOddvalues.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+int 
+main(){
+	int i, num[5], sum=0;
+	printf("\nInput the 1st number: "); 
+    scanf("%d", &num[0]);
```

Comment:
give proper tabs

take the number of input value from user and  use for loop for taking user inputs